imapbackup38.py

```diff
@@ -398,9 +398,10 @@ def print_usage():
     print (" -d DIR --mbox-dir=DIR         Write mbox files to directory. (defaults to cwd)")
     print (" -a --append-to-mboxes         Append new messages to mbox files. (default)")
     print (" -y --yes-overwrite-mboxes     Overwite existing mbox files instead of appending.")
-    print (" -f FOLDERS --folders=FOLDERS  Specifify which folders use.  Comma separated list.")
+    print (" -f FOLDERS --folders=FOLDERS  Specify which folders to include. Overrides --exclude-folders. Comma separated list.")
+    print (" --exclude-folders=FOLDERS     Specify which folders to exclude. Overridden by --folders. Comma separated list.")
```

Comment:
How does this PR support folders with commans in their names (I've seen such)?

Reply:
It doesn't, it behaves just like the existing --folders option and they should match each other. I hadn't thought about that case though. Some languages let you use any character to delimiter things like a regex which is nice if you need to match something with a bunch of slashes for example. Having a configurable delimiter might be a good way to solve it, maybe something like `--folder-delimiter='$' --folders='Inbox$Sent$Archive'`?

I feel like changing how --folders works is probably a separate issue/PR though.

Reply:
> I feel like changing how --folders works is probably a separate issue/PR though.

:+1: